curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-debugging-by-building-a-random-background-color-changer/66323433f931ca32305a11f5.md

```diff
@@ -0,0 +1,121 @@
+---
+id: 66323433f931ca32305a11f5
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+
+# --description--
+
+CamperBot is trying to build out a random background color changer. But they keep running into issues and need your help to debug the code. 
+
+CamperBot has created an array of dark colors and is trying to create a function that will return a random index from the array. But they have run into the following error message:
+
+```js
+Uncaught ReferenceError: math is not defined
+```
+
+A `ReferenceError` is thrown when a non-existent variable is referenced. In this case, it looks like CamperBot is trying to use `math` but JavaScript doesn't have a `math` object.
+
+Fix CamperBot's error in the `math.random()` line. Then open up the console to see the error message disappear.
```

Comment:
```suggestion
Fix CamperBot's error in the `math.random()` line. Then open up the console to see if the error message disappeared or not.
```

Reply:
I don't think we should change this to `if the error message disappeared or not.` Because that implies after making the change, the error message might still be there. But if done correctly, the error message shouldn't be there.

We could change this to be 

```
Fix CamperBot's error in the `math.random()` line and open up the console again.
```
Because the goal is to show campers that there is an error and get them in the habit of checking console statements. Then have them fix the error. and get them in the habit of checking the console again.